### DISCUSSION TITLE
Nexus Initialization Codes Can not Be Changed?

### DISCUSSION THREAD

**Discussion by adnanfuat** [prisma/prisma#2330] [Q&A] [answered]
Hi everyone. There is an issue with prisma-examples.

**STEP 1 )** I downloaded graphql-apollo-server from prisma-examples
I changed datasource information as http://prntscr.com/s8b5m6
**STEP 2 )** npm install and OKEY http://prntscr.com/s8b7rf
**STEP 3 )** npx prisma introspect and OKEY http://prntscr.com/s8b980
**STEP 4 )** npx prisma generate and OKEY http://prntscr.com/s8banz
**STEP 5 )** But now there is a problem. (Or I dont know true way)

schema.graphql's codes didnt change. // Same with initilization codes
http://prntscr.com/s8bbpm

and schema.ts file is same also.
http://prntscr.com/s8bcv0

How can I auto generate this two files.
Because of these problems, I cannot go forward.

Thanks for helps.

**Comment by ryands17** [comment, marked as answer]
Hey @adnanfuat :wave: 
For `schema.graphql` to change, you would have to make changes in `schema.ts` and run `npm run generate` again to see changes in your schema.

For e.g.

Considering your model `gorevlendirmeler`, you would need to create a type in your `schema.ts` as follows:

```ts
const gorevlendirmeler = objectType({
  name: 'gorevlendirmeler',
  definition(t) {
    t.model.baslik()
    t.model.firma()
  },
})
```

Now you can run `npm run generate` and the `schema.graphql` and the Nexus types will also change!

Let me know if that works for you.

**Reply by adnanfuat** [reply]
But. The point that I dont understand is why I write codes manually (const gorevlendirmeler = objectType({......)

Is there a way to create schema.ts automatically..
Or it is imperative to write codes of schema.ts

**Reply by ryands17** [reply]
Hey @adnanfuat :wave: 
That is currently not possible with Nexus. You would have to manually write in `schema.ts` to generate `schema.graphql`

Also you could use [this](https://www.npmjs.com/package/nexus-schema-prisma) library for generating easily from the schema.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
